docs/_edot-sdks/features.yml

```diff
@@ -276,8 +276,8 @@ features:
     is_sub_feature: true
     min_stack_version:
     .NET:
-      status: 
-      min_version: 
+      status: ga
+      min_version: .NET 9+
```

Comment:
I've marked this as supported with .NET 9+ because that's the first version to include built-in metrics conforming to the semantic conventions we helped define. Technically, runtime metrics are available for other runtimes using the contrib metrics instrumentation. I'm not sure if we need to call that out here.